react-graphql-apollo-tutorial.md

```diff
@@ -2929,13 +2927,13 @@ class Issues extends React.Component {
 }
 {{< /highlight >}}
 
-That's it. The client-side filtering should work for you. The button is used to toggle between the three states which is managed in the local state of the component. Only for two of these states the issues are queried, filtered and rendered. In the next step, the existing client-side filtering should be advanced to a server-side filtering which means that the filtered issues are already requested from the server and not filtered afterward on the client.
+You have implemented client-side filtering. The button is used to toggle between the three states managed in the local state of the component. Only in filtered and rendered states are the issues are queried. In the next step, the existing client-side filtering should be advanced to a server-side filtering, which means the filtered issues are already requested from the server and not filtered afterward on the client.
```

Comment:
"states are the issues are queried" to "states the issues are queried"

Reply:
"The issues are only queried in filtered and rendered states."

Reply:
Let's simply remove the complete sentence, maybe it adds too much confusion.

Remove: Only in filtered and rendered states are the issues are queried.